src/main/java/de/retest/recheck/report/TestReportFilter.java

```diff
@@ -42,7 +41,7 @@ static SuiteReplayResult filter( final SuiteReplayResult suiteReplayResult, fina
 		return newSuiteReplayResult;
 	}
 
-	static TestReplayResult filter( final TestReplayResult testReplayResult, final Filter filter ) {
+	public static TestReplayResult filter( final TestReplayResult testReplayResult, final Filter filter ) {
```

Comment:
If you think it makes sense, we can also rename the class to something like `FilterUtil` and make all methods `public`.

Reply:
Let's discuss this in a separate PR.